server/src/rbac.rs

```diff
@@ -105,7 +105,11 @@ impl Users {
     }
 
     pub fn remove_session(&self, session: &SessionKey) -> Option<String> {
-        mut_sessions().remove_session(session)
+        if !session.is_admin() {
+            mut_sessions().remove_session(session)
+        } else {
+            None
```

Comment:
The only place where result of this function is in logout use case. So this might casuse issue. Thinking should we move this check to a layer above. Somehow we need to keep the admin session alive throughout the life time of the parseable service